content/200-orm/200-prisma-client/300-client-extensions/500-middleware/index.mdx

```diff
@@ -67,7 +67,7 @@ The following sample scenarios show how to use middleware in practice:
 
 ## Where to add middleware
 
-Add Prisma middleware **outside the context of the request handler**, otherwise each request adds a new _instance_ of the middleware to the stack. The following example demonstrates where to add Prisma middleware in the context of an Express app:
+Add Prisma ORM middleware **outside the context of the request handler**, otherwise each request adds a new _instance_ of the middleware to the stack. The following example demonstrates where to add Prisma ORM middleware in the context of an Express app:
```

Comment:
```suggestion
Add Prisma Client middleware **outside the context of the request handler**, otherwise each request adds a new _instance_ of the middleware to the stack. The following example demonstrates where to add Prisma Client middleware in the context of an Express app:
```